plugins/scaffolder-backend/src/deprecated.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  ActionContext as ActionContextNode,
+  createTemplateAction as createTemplateActionNode,
+  TaskSecrets as TaskSecretsNode,
+  TemplateAction as TemplateActionNode,
+} from '@backstage/plugin-scaffolder-node';
+import { JsonObject } from '@backstage/types';
+
+/**
+ * @public
+ * @deprecated Use `ActionContext` from `@backstage/plugin-scaffolder-node` instead
```

Comment:
```suggestion
 * @deprecated Import from {@link @backstage/plugin-scaffolder-node#ActionContext} instead
```

Used something like this before, not sure if it make it easier or not.